varnish/datadog_checks/varnish/varnish.py

```diff
@@ -1,19 +1,19 @@
-# (C) Datadog, Inc. 2010-2017
+# (C) Datadog, Inc. 2018
 # All rights reserved
-# Licensed under Simplified BSD License (see LICENSE)
+# Licensed under a 3-clause BSD style license (see LICENSE)
 
 # stdlib
 from collections import defaultdict
+from distutils.version import LooseVersion
 from os import geteuid
-from distutils.version import LooseVersion # pylint: disable=E0611,E0401
+import json
 import re
-import xml.parsers.expat # python 2.4 compatible
 import shlex
-import simplejson as json
+import xml.parsers.expat  # python 2.4 compatible
```

Comment:
kill this with fire

Reply:
What do I use instead ? It is used to parse the output of varnish